boards/arm/nrf54h20pdk_nrf54h20/nrf54h20pdk_nrf54h20_cpuapp.dts

```diff
@@ -93,6 +96,32 @@
 	status = "okay";
 };
 
+&cpuapp_bellboard {
+	interrupts = <96 NRF_DEFAULT_IRQ_PRIORITY>;
+	interrupt-names = "irq0";
+	/* irq0: 0: cpuapp-cpusec, 13: cpuapp-cpuppr, 18: cpuapp-cpurad */
+	nordic,interrupt-mapping = <0x00042001 0>;
+};
+
+&cpusec_cpuapp_ipc {
```

Comment:
I understand the reason to avoid repeating code and reusing parts of this node between both peers in communication, but the result is a hard-to-understand content of this node because it is scattered across multiple files. It's hard to follow which mbox instance is tx and which is rx.

Reply:
@57300

Reply:
I was betting on the mbox placement being easy to deduce from the IPC node label. If it's `cpuxxx_cpuyyy_ipc`, then the first one is `cpuxxx_bellboard/vevif` and the second is `cpuyyy_bellboard/vevif`. If you're building for `cpuxxx`, then the first one is RX and the second is TX (to `cpuyyy`), and vice versa for the other core.

In addition, it should also be possible to tuck away the shared memory configuration in the common file, if we deprecate the `tx/rx-region` properties in favor of `memory-regions` and `memory-region-names`.

This information is not scattered just for the sake of avoiding repetition. It makes sense to track channel and memory configuration at system level, as opposed to individual core level.

But if that's too confusing, then I don't mind going back to duplicating nodes.